spec/app/lib/bing_api_v7_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe BingApiV7 do
+  describe '#search', vcr: true do
+    context 'with valid response data' do
+      let!(:company) { Company.create(domain: 'softserve') }
+      let!(:bing) { BingApiV7.new(company) }
+      subject { bing.search }
+
+      it 'returns json parsed data' do
+        expect(subject).to be_a(Hash)
+        expect(subject).not_to be_empty
+        expect(subject).to have_key('webPages')
+        expect(subject['webPages']).to have_key('value')
+        expect(subject["webPages"]["value"]).to be_a_kind_of(Array)
+      end
+    end
+  end
+
+  describe '#pages_process', vcr: true do
```

Comment:
put vcr tracking only for contexts where you use it to avoid unnecessary metadata like `vcr: false`